test/test_issue325.js

```diff
@@ -0,0 +1,43 @@
+
+exports.test = function(sql, assert){
+    "use strict";
+    // Create a database
+    var db = new sql.Database();
+    var result;
+    var value;
+
+    // inline result value test
+    result = db.exec("SELECT 1.7976931348623157e+308 AS value");
+    value = result[0].values[0][0];
+    assert.strictEqual(value, 1.7976931348623157e+308);
+
+    // bind result value test
+    result = db.exec("SELECT ? AS value", [1.7976931348623157e+308]);
+    value = result[0].values[0][0];
+    assert.strictEqual(value, 1.7976931348623157e+308);
+
+    // CAST value test
+    result = db.exec("SELECT CAST('1.7976931348623157e+308' AS REAL) AS value");
```

Comment:
What is the purpose of this test ? If `"SELECT 1.7976931348623157e+308 AS value"` passes, then this one will pass too, won't it ? And if one of them passes and not the other, it will be a bug it sqlite itself, not sql.js.

Reply:
as a non-system-programmer who doesn't understand sqlite-internals, i'm paranoid/suspicious CAST might have its own parsing logic/magic.  better to be safe and verify it works as well.

here, unpatched demo shows CAST as Infinity:
![image](https://user-images.githubusercontent.com/280571/78459649-25c85980-7680-11ea-814e-809e17c0f541.png)

Reply:
Well, cast can work however it wants, it's not in the scope of sql.js to test it. In the semantics of SQLite `select 1.7976931348623157e+308` and `select cast('1.7976931348623157e+308' as REAL)` is the same, so we don't need to test both.

Reply:
ok, marking as resolved with latest commit.